cpp/src/aztec/dsl/acir_format/merkle_insert.hpp

```diff
@@ -0,0 +1,39 @@
+#include <stdlib/types/types.hpp>
+
+namespace acir_format {
+
+struct MerkleInsertConstraint {
```

Comment:
We can get rid of this file now as its no longer being used as an opcode and if brought back, we can do it in Noir

Reply:
👍  Removed!